audio/wave_shaper_node.rs

```diff
@@ -79,33 +93,68 @@ impl AudioNodeEngine for WaveShaperNode {
         AudioNodeType::WaveShaperNode
     }
 
-    fn process(&mut self, mut inputs: Chunk, _info: &BlockInfo) -> Chunk {
+    fn process(&mut self, mut inputs: Chunk, info: &BlockInfo) -> Chunk {
         debug_assert!(inputs.len() == 1);
 
         if inputs.blocks[0].is_silence() {
```

Comment:
@Manishearth could you confirm whether tail-time here could be implemented like this:
if we hit two silent blocks in row after non-silent block, we `explicit_silence()` them and run them through the waveshaping so all the tail-time from resampler would be spitted out

Reply:
That sounds correct.